lib/plottr_components/src/components/timeline/TimelineTable.js

```diff
@@ -57,6 +64,12 @@ const TimelineTableConnector = (connector) => {
 
     componentDidUpdate() {
       this.setLength()
+
+      if (this.state.toastProps.visible) {
```

Comment:
This is the wrong place for this timeout to be registered -- what happens when there are more updates before the timer expires?
If the state is in `redux`, this should all become clearer.